ebb_lint/test/test_flake8.py

```diff
@@ -400,6 +400,30 @@ class C(object):
 $L204$'a' 'b'  $L303$# noqa
 
     ''',
+
+    # Oh boy these following two cases need explanation. The prefix of the
+    # 'spam' node in both of these cases ends with a literal \ followed by a
+    # newline, which cannot be tokenized by generate_tokens because it expects
+    # something to follow that sequence. This is one hell of a special case,
+    # but it made flake8 fail hard on valid syntax. The first case is a prefix
+    # containing no comment, and the second case also includes a comment which
+    # requires scanning.
```

Comment:
No. The assignment doesn't matter; it's just that the string literal node's prefix ends with `'\\\n'`, and a string passed to `generate_tokens` can't end with that. A node's prefix is every non-token character before the node, such as whitespace, comments, and line continuation characters.